com.vscodium.codium.yaml

```diff
@@ -0,0 +1,83 @@
+default-branch: stable
+base: org.electronjs.Electron2.BaseApp
+base-version: '19.08'
+runtime: org.freedesktop.Sdk
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: codium
+desktop-file-name-suffix: ' (Flatpak)'
+separate-locales: false
+finish-args:
+  - --require-version=0.10.3
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --socket=ssh-auth
+  - --share=network
+  - --device=all
+  - --filesystem=host
+  - --allow=devel
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.freedesktop.Flatpak
+  - --env=NPM_CONFIG_GLOBALCONFIG=/app/etc/npmrc
+  - --env=LD_LIBRARY_PATH=/app/lib
+  - --filesystem=xdg-config/kdeglobals:ro
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=com.canonical.AppMenu.Registrar.*
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /share/gtk-doc
+  - '*.la'
+modules:
+  - shared-modules/libsecret/libsecret.json
+  - name: codium
+    buildsystem: simple
+    build-commands:
+      - install -D codium.sh /app/bin/codium
+      - install -Dm644 codium_64.png /app/share/icons/hicolor/64x64/apps/com.vscodium.codium.png
+      - install -Dm644 codium_128.png /app/share/icons/hicolor/128x128/apps/com.vscodium.codium.png
+      - install -Dm644 codium_256.png /app/share/icons/hicolor/256x256/apps/com.vscodium.codium.png
+      - install -Dm644 codium_512.png /app/share/icons/hicolor/512x512/apps/com.vscodium.codium.png
+      - install -Dm644 codium.svg /app/share/icons/hicolor/scalable/apps/com.vscodium.codium.svg
+      - install -Dm644 com.vscodium.codium.appdata.xml -t /app/share/metainfo
+      - install -Dm644 com.vscodium.codium.desktop -t /app/share/applications
+      - install -Dm644 npmrc -t /app/etc
+      - cp /usr/bin/ar /app/bin
+      - ARCH_TRIPLE=$(gcc --print-multiarch) && cp /usr/lib/${ARCH_TRIPLE}/libbfd-*.so
+        /app/lib
+      - ARCH_TRIPLE=$(gcc --print-multiarch) && ln -s /usr/lib/${ARCH_TRIPLE}/libtinfo.so/app/lib/libtinfo.so.5
+      - ar x codium.deb
+      - tar xf data.tar.xz
+      - mv usr/share/codium /app/share
+      - rm -r codium.deb control.tar.gz data.tar.xz debian-binary usr
+      - install -Dm644 flatpak-warning.txt -t /app/share/codium
+    sources:
+      - type: file
+        path: codium.sh
+      - type: file
+        url: https://github.com/VSCodium/vscodium/releases/download/1.43.2/codium_1.43.2-1585083818_amd64.deb
+        sha256: 3e5f307d1321d65809d60cf61f9955fa1ac83021c157bc727cdd6485cd209cb4
+        dest-filename: codium.deb
+      - type: file
+        url: https://raw.githubusercontent.com/VSCodium/vscodium/master/src/src/vs/workbench/contrib/update/browser/media/code-icon.svg
+        sha256: 4b0b8f198fda18a79bfd293f6ef7ba3db167f35324c97cbafda881840be8a981
+        dest-filename: codium.svg
+      - type: file
+        path: flatpak-warning.txt
+      - type: file
+        path: npmrc
+      - type: file
+        path: com.vscodium.codium.appdata.xml
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
you mean I need to rename com.vscodium.codium.appdata.xml to com.vscodium.codium.metainfo.xml?

Reply:
Yes